### ISSUE TITLE
C Stack trace after upgrade

### ISSUE THREAD

**Issue by hahaxd35** [ONLYOFFICE/DocumentServer#989]
**Do you want to request a *feature* or report a *bug*?**
BUG

**What is the current behavior?**
unable to save file 

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
upgrade from 5.6.2 or 5.5.1 to 5.6.4

**What is the expected behavior?**

**Did this work in previous versions of DocumentServer?**
yes  5.5.1
**DocumentServer version:**
5.6.4

**Operating System:**
debian 10 

**Browser version:**
------

Errors:
`Error: /var/www/onlyoffice/documentserver/server/FileConverter/bin/x2t exited with signal: SIGABRT
    at ChildProcess.completionListener (/snapshot/server/build/server/FileConverter/node_modules/@expo/spawn-async/build/spawnAsync.js:51:23)
    at Object.onceWrapper (events.js:286:20)
    at ChildProcess.emit (events.js:198:13)
    at maybeClose (internal/child_process.js:982:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
    ...
    at spawnAsync (/snapshot/server/build/server/FileConverter/node_modules/@expo/spawn-async/build/spawnAsync.js:17:21)
    at ExecuteTask (/snapshot/server/build/server/FileConverter/sources/converter.js:0:0)
    at ExecuteTask.next (<anonymous>)
    at /snapshot/server/build/server/FileConverter/sources/converter.js:0:0
    at Generator.next (<anonymous>)
    at onFulfilled (/snapshot/server/build/server/FileConverter/node_modules/co/index.js:65:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)
[2020-09-10T18:51:15.672] [ERROR] nodeJS - stdout (id=xsVSDeiKgeILPiQucX3G_2089):
[2020-09-10T18:51:15.672] [ERROR] nodeJS - stderr (id=xsVSDeiKgeILPiQucX3G_2089):terminate called without an active exception
Received signal 6

==== C stack trace ===============================

 [0x7fc58e17dd82]
 [0x7fc58d7c9730]
 [0x7fc58d4a87bb]
 [0x7fc58d493535]
 [0x7fc5902e3a75]
 [0x7fc5902c1bc6]
 [0x7fc5902c1bf3]
 [0x7fc5902c18c6]
 [0x0000018e1e0b]
 [0x000001739c0a]
 [0x00000173a48d]
 [0x0000009f9728]
 [0x0000009ffe8d]
 [0x000000a012f4]
 [0x0000016bac75]
 [0x0000016baeb2]
 [0x0000016e216d]
 [0x00000151e492]
 [0x00000072d24d]
 [0x000000734c45]
 [0x0000007380b6]
 [0x00000053cbbf]
 [0x7fc58d49509b]
 [0x000000713ca3]
[end of stack trace]

[2020-09-10T18:51:15.673] [ERROR] nodeJS - ExitCode (code=0;signal=SIGABRT;error:-80;id=xsVSDeiKgeILPiQucX3G_2089)

`

GUI is loading you can work no problems. Saving not possible. Another Server who will download the files "says" the callback download Links was not send.

Can you help me?

**Comment by ShockwaveNN** [comment]
Hi, we aware for this issue (issue #46499 in our private bug tracker)

But as far as we know - this issue actual only on small amount docx files (with empty `<a:solidFill/>`)

Did this problem actual for you in any file? Or in some specific?

**Comment by hahaxd35** [comment]
With an .ods file there were no errors/problems.

Unfortunately we save all files in .docx 
Each file is therefore not storable.

You knew about the error and you can still install it? How can i downgrade to 5.6.2? we use onlyoffice-documentserver-de

**Comment by ShockwaveNN** [comment]
> Each file is therefore not storable.

To be clear - you create new empty docx file, add some info and save it as docx and after that very is failure? Could you describe your scenario step by step/

> You knew about the error and you can still install it?

We think this problem actual for very little user base, but with your report I need more info to analyze it

>  How can i downgrade to 5.6.2? we use onlyoffice-documentserver-de

Sorry, we do not store old version of deb packages in our repo (see here https://github.com/ONLYOFFICE/DocumentServer/issues/319)
But you can install older docker version, or extract deb version from onlyoffice/documentserver-de:5.6.2 docker using `dpkg-repack`, but not 100% sure it's working version, I'll try to search for original one

**Comment by hahaxd35** [comment]
> To be clear - you create new empty docx file, add some info and save it as docx and after that very is failure? Could you describe your scenario step by step/

I create i emtpy .docx File. Edit this file and save -> this works. After saving and edit again the server cant save this file again. 

If i upload a .ods file i can edit this file and save. The saved file will be stored as .docx. 
This .docx i cant save again.

C Stack Trace  in log

**Comment by ShockwaveNN** [comment]
> I create i emtpy .docx File. Edit this file and save -> this works. After saving and edit again the server cant save this file again.

Can you record a video with problem, still not able to reproduce this with empty docx file

**Comment by hahaxd35** [comment]
emtpy file saving works but after converting (by ONLYOFFICE) saving again doesn't work

**Comment by ShockwaveNN** [comment]
> emtpy file saving works but after converting (by ONLYOFFICE) saving again doesn't work

Yeah, I understand you right, but still can't reproduce. Please record a video

**Comment by hahaxd35** [comment]
https://mirror.digionline.de/digi/ONYLOFFICETEST-ods-and-empty.mp4

**Comment by ShockwaveNN** [comment]
Yeah, I see, scenario seems same as I use
But I use integrated test example
Could you repoduce it using test example?

Are you using some non-default value in your integration to your DMS, like `forcesave`?

You can look at my video:
[simplescreenrecorder-2020-09-11_13.45.48.zip](https://github.com/ONLYOFFICE/DocumentServer/files/5208438/simplescreenrecorder-2020-09-11_13.45.48.zip)

**Comment by hahaxd35** [comment]
'autosave' => true,
'forcesave' => false,

Know i have downgrade to 5.5.1 and now  i want to upgrade to 5.6.0 because one server got the old .deb file

**Comment by ShockwaveNN** [comment]
>'autosave' => true,
'forcesave' => false,

Yeah, those looks like default values

> now i want to upgrade to 5.6.0 because one server got the old .deb file

As I said earlier - currently I cannot find a way to get older deb files, except using `dpkg-repack` in docker

> we delete all onlyoffice-documetserver-example files ;-)

I was able to reproduce this problem, seems this problem actual only if starting from odt file
Via test example: 

1. Create new docx file and open in Document Editor
2. Enter `test` to first paragraph and save file as odt
3. Load this odt file to test example and open in DocumentEditor|
4. Add second paragraph with Test, save file and close it
5. Wait for one minute to file build and reopen it again
6. Add third paragraph with test, save file and close it

After step 6 there is error with SIGABRT in logs

**Comment by hahaxd35** [comment]
on our server this also happen when starting with .docx.
on one server there was a .deb file from /var/cache/apt/ with Version 5.6.0 and this Version is working.

Do you want this?

**Comment by ShockwaveNN** [comment]
> Do you want this?

Yeah, this would be great if you upload it.
I'm not sure what a lot people able to find this bug, but until we release bugfix and\or decide that to do with this issue - if someone found out this issue - they should downgrade

**Comment by ShockwaveNN** [comment]
We founf link to previous stable version

http://repo-doc-onlyoffice-com.s3-eu-west-1.amazonaws.com/ubuntu/trusty/onlyoffice-documentserver-de/hotfix/v5.6.2/5.6.2-2/repo/onlyoffice-documentserver-de_5.6.2-2_amd64.deb

**Comment by hahaxd35** [comment]
Here is 5.6.0
https://mirror.digionline.de/digi/onlyoffice-documentserver-de_5.6.0-17_amd64.zip

**Comment by orandev** [comment]
Hello
I have the same issue on my platform.
Servers are Ubuntu 18.04.
Nextcloud 19.0.2 on one server (onlyoffice app version 6.0.0).
OnlyOffice Document Server 5.6.4-20 on another server (+ external rabbitmq and redis and postgres).

When I edit one of the default files that are created by Nextcloud for each new user account (Documents > Welcome to Nextcloud Hub.docx), after I close the document, the following error is logged in Nextcloud:

"Track without url: 912 status 3"

And the following one in DS converter out.log:

[ERROR] nodeJS - error spawnAsync(id=3402183177)
Error: /var/www/onlyoffice/documentserver/server/FileConverter/bin/x2t exited with signal: SIGABRT
    at ChildProcess.completionListener (/snapshot/server/build/server/FileConverter/node_modules/@expo/spawn-async/build/spawnAsync.js:51:23)
    at Object.onceWrapper (events.js:286:20)
    at ChildProcess.emit (events.js:198:13)
    at maybeClose (internal/child_process.js:982:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
    ...
    at spawnAsync (/snapshot/server/build/server/FileConverter/node_modules/@expo/spawn-async/build/spawnAsync.js:17:21)
    at ExecuteTask (/snapshot/server/build/server/FileConverter/sources/converter.js:0:0)
    at ExecuteTask.next (<anonymous>)
    at /snapshot/server/build/server/FileConverter/sources/converter.js:0:0
    at Generator.next (<anonymous>)
    at onFulfilled (/snapshot/server/build/server/FileConverter/node_modules/co/index.js:65:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)
[ERROR] nodeJS - stdout (id=3402183177_5274):
[ERROR] nodeJS - stderr (id=3402183177_5274):terminate called without an active exception
Received signal 6

==== C stack trace ===============================

 [0x7f19a6da5d82]
 [0x7f19a61fb8a0]
 [0x7f19a5a98f47]
 [0x7f19a5a9a8b1]
 [0x7f19a8f09a75]
 [0x7f19a8ee7bc6]
 [0x7f19a8ee7bf3]
 [0x7f19a8ee78c6]
 [0x0000018e1e0b]
 [0x000001739c0a]
 [0x00000173a48d]
 [0x0000009f9728]
 [0x0000009ffe8d]
 [0x000000a012f4]
 [0x0000016bac75]
 [0x0000016baeb2]
 [0x0000016e216d]
 [0x00000151e492]
 [0x00000072d24d]
 [0x000000734c45]
 [0x0000007380b6]
 [0x00000053cbbf]
 [0x7f19a5a7bb97]
 [0x000000713ca3]
[end of stack trace]

[ERROR] nodeJS - ExitCode (code=0;signal=SIGABRT;error:-80;id=3402183177_5274)


And, indeed, this default Nextcloud docx file has an empty "<a:solidFill></a:solidFill>" node inside it.

**Comment by ShockwaveNN** [comment]
@orandev Hello, sorry for this error.
We fixed it in
https://github.com/ONLYOFFICE/sdkjs/commit/d1c2265f153e8eb43a9f6970cb080d5967f40f12
https://github.com/ONLYOFFICE/sdkjs/commit/a6d5cf09c0f2d638b3cee5cd02a2b272c8c908b2

and hope to release those fixes as version 5.6.5 in several days

**Comment by ShockwaveNN** [comment]
DocumentServer v5.6.5 is released and this issue should be fixed

**Comment by stur86** [comment]
Is this going to be pushed to the Ubuntu repository soon? It still has v5.6.4 and I just had this same problem (lost quite a bit of work to it!).

**Comment by ShockwaveNN** [comment]
@stur86 could you check again, we released 5.6.5 in our official repo around 2 weeks ago, did you apt-get update repo?

**Comment by stur86** [comment]
I was referring to the default version that appears in the Ubuntu Software Centre (from the Snap Store), I guess I need to add your repo? Anyway I'll try updating.

**Comment by ShockwaveNN** [comment]
@stur86 Snap of DocumentServer are not our high-priority project, so we do not update it regularly, and have some issues with it. We advice you to use Docker or deb installation

**Comment by stur86** [comment]
The .deb package on https://www.onlyoffice.com/en/download-desktop.aspx is still at 5.6.4 too though.

**Comment by ShockwaveNN** [comment]
@stur86 Yes, this is normal. This issue for DocumentServer and as far as we aware this problem is not actual for DesktopEditors

**Comment by stur86** [comment]
It happened in the Desktop Editor for me though.

**Comment by ShockwaveNN** [comment]
@stur86 Could you create new issue for DesktopEditor with more details?

**Comment by stur86** [comment]
Just done that, let's move the discussion there.

**Comment by jrjena136** [comment]
@ShockwaveNN could you please help. I was also getting that error. But I am now using the latest image `onlyoffice/documentserver:latest`. I am running through this issue.
```
==> /var/log/onlyoffice/documentserver/converter/out.log <==
[2020-11-04T11:05:37.989] [ERROR] nodeJS - stdout (id=41b302cd1_4443):
[2020-11-04T11:05:37.990] [ERROR] nodeJS - stderr (id=41b302cd1_4443):change_code: a)+1]=d};Tb.prototype.ZI=function(){this.Na&&(this.Na.gA.pue(),this.Na.ud.Lgb(Ff))};Tb.prototype.Mm=function(c){if(this.Na){for(var a=0;a<this.aa.length&&(!this.aa[a].xb||this.aa[a].xb()!==c);++a);if(a<this.aa.length)return this.Na.Mm(this.xb())}return!1};Tb.prototype.J0=function(){return this.Mm(void 0)};Tb.prototype.Sq=function(){this.Na&&this.Na.Ba&&this.Na.Ba.pxb&&this.Na.Ba.lr.Fvb(this.Na)};Tb.prototype.rFa=function(){return!0};Tb.prototype.Pv=function(){};Tb.prototype.Jnb=function(){this.Na&&
change: Uncaught TypeError: Cannot read property 'pue' of undefined
DoctRenderer:<result><error index="8" /></result>

[2020-11-04T11:05:37.990] [ERROR] nodeJS - ExitCode (code=86;signal=null;error:-86;id=41b302cd1_4443)
[2020-11-04T11:11:23.926] [ERROR] nodeJS - stdout (id=5897a4e21_5887):
[2020-11-04T11:11:23.926] [ERROR] nodeJS - stderr (id=5897a4e21_5887):change_code: a)+1]=d};Tb.prototype.ZI=function(){this.Na&&(this.Na.gA.pue(),this.Na.ud.Lgb(Ff))};Tb.prototype.Mm=function(c){if(this.Na){for(var a=0;a<this.aa.length&&(!this.aa[a].xb||this.aa[a].xb()!==c);++a);if(a<this.aa.length)return this.Na.Mm(this.xb())}return!1};Tb.prototype.J0=function(){return this.Mm(void 0)};Tb.prototype.Sq=function(){this.Na&&this.Na.Ba&&this.Na.Ba.pxb&&this.Na.Ba.lr.Fvb(this.Na)};Tb.prototype.rFa=function(){return!0};Tb.prototype.Pv=function(){};Tb.prototype.Jnb=function(){this.Na&&
change: Uncaught TypeError: Cannot read property 'pue' of undefined
DoctRenderer:<result><error index="8" /></result>

[2020-11-04T11:11:23.926] [ERROR] nodeJS - ExitCode (code=86;signal=null;error:-86;id=5897a4e21_5887)
```
